### ISSUE TITLE
When adding affiliate via New User screen, automatically approve them

### ISSUE THREAD

**Issue by zackkatz** [AffiliateWP/AffiliateWP#2090]
It seems that if I add an affiliate in the New User screen, they should be auto-approved. Any reason for this not to be the case?

Related to #1144

**Comment by pippinsplugins** [comment]
@zackkatz it's supposed to set the status based on the settings you have in Misc. Do you require affiliates to be approved during normal registration?

**Comment by zackkatz** [comment]
Yes, but my thinking is that manually adding the affiliate is already "approving" them.
